var/spack/repos/builtin/packages/singularity/package.py

```diff
@@ -26,6 +26,7 @@ class Singularity(MakefilePackage):
 
```

Comment:
@ArangoGutierrez If you're a singularity developer, you can add the following:
```python
maintainers = ['ArangoGutierrez']
```
That way, if someone runs into problems with `singularity`, or wants to modify the package, we can ping you and let you know.

Reply:
7b01f38